libs/pmp-web/user/single-user-statistics/feature/src/lib/containers/single-user-statistics/single-user-statistics.component.ts

```diff
@@ -34,6 +34,10 @@ export class SingleUserStatisticsComponent implements OnInit, OnDestroy {
     this.initSubscribeUserStatistics();
   }
 
+  onNavigateItem(prStatistics: PrStatistics): void {
+    window.open(prStatistics.url, '_blank');
```

Comment:
Normally we should use the Angular router to navigate to another view, but I am not sure what where requirements here, do statistics must be open in the new window?

Reply:
That logic is already implemented at master, I merged that with my changes where I moved out the single-user-statistics table to shared ui lib

Reply:
As this url point to another application (in this case to github) we can't use angular navigation service.